.github/workflows/quarkus.yml

```diff
@@ -37,8 +37,17 @@ env:
 
 jobs:
   build-mandrel:
-    name: Mandrel build
+    name: Mandrel build - ${{ matrix.jdk }}
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: ['OpenJDK-latest-GA', 'OpenJDK-latest-EA']
```

Comment:
`OpenJDK-11-latest-GA` ? `OpenJDK-latest-GA` would be 14 now and 15 mid Sept.

Reply:
Good catch